src/ydata_synthetic/postprocessing/timeseries/inverse_preprocesser.py

```diff
@@ -0,0 +1,17 @@
+from typing import Union, List
+
+from ydata_synthetic.postprocessing.regular import inverse_preprocesser
+
+from sklearn.pipeline import Pipeline
+from sklearn.compose import ColumnTransformer
+from sklearn.preprocessing import PowerTransformer, OneHotEncoder, StandardScaler, MinMaxScaler
+
+from pandas import DataFrame
+
+def inverse_transform(data: List, processor: Union[Pipeline, ColumnTransformer, PowerTransformer, OneHotEncoder,
```

Comment:
Why is the data of type list?

Reply:
The preprocessing that was leveraged from the processed stock was returning a list of rolled sequences as numpy arrays.